sentry-uitest/sentry-uitest-android/src/main/java/io/sentry/uitest/android/utils/BooleanIdlingResource.kt

```diff
@@ -0,0 +1,30 @@
+package io.sentry.uitest.android.utils
+
+import androidx.test.espresso.IdlingResource
+import java.util.concurrent.atomic.AtomicBoolean
+
+/** Idling resource based on a boolean flag. */
+class BooleanIdlingResource(private val name: String) : IdlingResource {
```

Comment:
I think this one is unused in this module?

Reply:
Not at the moment, but since new tests will be added it's something that could make sense to keep.
Can remove it, then just copy again from the benchmark module in case it's needed. Or just keep it here 🤔

Reply:
Maybe put it into `sentry-test-support` so it can be shared across modules?

Reply:
BooleanIdlingResource extends Espresso's `IdlingResource`
Is it fine to add an Android specific class to `sentry-test-support`, as it's used by plain java sdk, too?
If yes, i'll add it there

Reply:
mmm, I'd say it's fine, since it's a test dependency anyway.. but maybe @marandaneto has a different opinion - we could create a new module `sentry-android-test-support` in the worst case.

Reply:
Since it's testing only it's fine, the question is if the `java` tolling won't complain when trying to build it.
Otherwise, we can create a `sentry-android-test-support` as @romtsn said.

Reply:
I tried to create a new module `sentry-android-test-support`
There are several things, though. The most important thing is that the `BooleanIdlingResource` is needed as a normal dependency, not a test dependency. I think creating a whole module just for one class is an overkill.
So i make `sentry-uitest-android-benchmark` depending on `sentry-uitest-android`, so that only `sentry-uitest-android` contains BooleanIdlingResource
Is it fine? we could change it later on, if more things get added